### DISCUSSION TITLE
What is the way to document known issues in release notes

### DISCUSSION THREAD

**Discussion by iViggyPrabhu** [CycloneDX/specification#145] [Q&A] [answered]
Release notes generally do include known issues in the release. I dont know if it is right to classify it in Vulnerabilities as it is not necessary a security vulnerability, maybe just a bug or some feature that has not been fully implemented. I was not able to find a way to capture it in the specification. There is [resolves in release notes](https://cyclonedx.org/docs/1.4/json/#components_items_releaseNotes_resolves) to document what have been resolved.

**Comment by stevespringett** [comment, marked as answer]
Release notes, in their current form, do not directly support "known issues". The only way to accomplish that today is to include them in the note itself. The team is currently planning v1.5 of the spec, so if this something you'd like to see, I'd encourage you to create an enhancement request and the team will evaluate.

Thanks for the suggestion.

**Reply by iViggyPrabhu** [reply]
Thanks @stevespringett . I will look into the process of submitting an enhancement request.
